README.md

```diff
@@ -49,14 +40,13 @@ already comfortable with `tasty-quickcheck` and property testing in general; for
 those who are not, or those who prefer to learn from examples, we provide three
 executable examples:
 
-* `examples/square-tests`, which demonstrates basic usage of both universal and
-  conditional properties;
-* `examples/zip-tests`, which demonstrates conditional properties with crashing
-  cases; and
-* `examples/natural-tests`, which demonstrates law checking.
+* `examples/reverse-tests`, which demonstrates how to define property 
+using definition of behavior in Haskell. 
+* `examples/zip-tests`, which demonstrates how to use `fromPFun` to 
+define Plutarch property. 
 
 Users new to the library can read the examples as code, as well as execute the
-tests with `cabal new-test` to see what they do.
+tests with `cabal *-test` to see what they do.
```

Comment:
We can probably just say `cabal test`: the old-style versions are long gone now.